SPECS-EXTENDED/nodejs-nodemon/nodejs-nodemon.spec

```diff
@@ -8,11 +8,11 @@ Distribution:   Mariner
 
 Name:          nodejs-%{npm_name}
 Version:       2.0.3
-Release:       2%{?dist}
+Release:       3%{?dist}
 Summary:       Simple monitor script for use during development of a node.js app
 License:       MIT
 URL:           https://github.com/remy/nodemon
-Source0:       %{npm_name}-v%{version}-bundled.tar.gz
+Source0:       https://github.com/remy/%{npm_name}/archive/v%{version}.tar.gz
```

Comment:
I see two issues here:
1. The file from the provided URL has a different hash, than what we're building with. If you verified there are no differences between the contents and that the package builds fine with the new source, then this is ideal and let's take it. Otherwise, let's note down where the source is coming from and use our blob storage for now - the main goal here is to provide valid path to the sources our packages use in the CGmanifest.
2. This changes the name of the source tarball. If you're going to proceed with the new source, it can be easily adjusted by appending `#/[actual_tarball_name]` to the URL.

Reply:
Thank you for highlighting these issues. I have updated the spec files according to this suggestion.